sycl/doc/extensions/supported/sycl_ext_oneapi_bfloat16.asciidoc

```diff
@@ -106,16 +89,18 @@ enum class aspect {
 If a SYCL device has the `ext_oneapi_bfloat16` aspect, then it natively
 supports conversion of values of `float` type to `bfloat16` and back.
```

Comment:
OK. Just to check whether we agree on the meaning of this aspect, it will report _true_ if native or fallback support exists for bfloat16.
Then, once fallback is implemented, the L0 plugin will always return true for this aspect and will use the underlying device query to decide whether it needs to use the fallback library.